woocommerce/woocommerce-android
# [Woo POS][Cash & Receipts] Order completion call. Navigation to the payment success

Closes: #13084


### Description


### Testing information

* More -> POS -> New order
* Choose pay with cash
* Complete the order
* Notice that success screen is shown
* Go to the order list in the main app. Notice new, complete order. Check it detals

Try back navigation from the cash payment screen, too

### The tests that have been performed

Above

### Images/gif




https://github.com/user-attachments/assets/346bb788-d5e9-4100-b5c2-af3695da9d04






## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
